tests/test_using.py

```diff
@@ -1137,6 +1137,24 @@ class Meta:
         obj = TestObjectFactory.build()
         self.assertEqual(42, obj.attributes)
 
+    def test_rename_non_existent_kwarg(self):
+        '''
+        If a key is specified in the rename attribute of the model factory, but
+        not passed into the object when instantiated we should bypass the
+        keyword re-assignment instead of throwing an error.
+        https://github.com/FactoryBoy/factory_boy/issues/504
+        '''
```

Comment:
I think the test is clear enough so that this docstring is mostly paraphrasing it. I think it would be good to reference the issue in the commit message, so that this information is available on `git blame`.

Reply:
Sounds good @francoisfreitag . I will push an extra commit resolving all other feedback and then I can squash it down to a single commit with a reference to the issue number.